src/server/helpers/achievement.js

```diff
@@ -113,7 +118,10 @@ function awardTitle(editorId, tier) {
 						const out = {};
 						out[tier.titleName] = unlock;
 						return out;
-					});
+					})
+					.catch((err) =>
+						console.log(err)
```

Comment:
Hmm, is there no way to detect the editor ID error and report it by throwing a different kind of error?